### PR TITLE
#34 Delete entry

### LINKED ISSUE #34: Delete entry.
:information_source: This issue is part of the Epic #12 .

### Objective
Create a module to delete an entry from the Tickspot API.

Documentation:
- [Delete entry](https://github.com/tick/tick-api/blob/master/sections/entries.md#delete-entry).

### CheckList
- [ ] Create a method to delete an entry.
- [ ] Create tests.

### PR DESCRIPTION
**Pull request by MrJuanGaviriaK**
Closes #34 

:information_source: This issue is part of the Epic #12 .

### Objective
Create a module to delete an entry from the Tickspot API.

### CheckList
- [x] Create a method to delete an entry.
- [x] Create tests.

### CODE REVIEW

File: test/v2/getEntry.test.js [hunk 1]
```diff
@@ -12,10 +12,7 @@ const getEntryUrl = 'https://www.tickspot.com/123456/api/v2/entries/123456.json'
 describe('getEntry', () => {
   describe('when API call is successful', () => {
     const responseData = responseFactory({}, 'successful', dataEntrySuccessful, getEntryUrl);
-
-    beforeEach(() => {
-      axios.get.mockResolvedValueOnce(responseData);
-    });
+    axios.get.mockResolvedValueOnce(responseData);
```
Comment (brayvasq) [inline test/v2/getEntry.test.js:15]: @MrJuanGaviriaK  Why did you remove this line from the beforeEach block?

**Review by brayvasq — COMMENTED** [review]
:wave:  Hi @MrJuanGaviriaK Thanks for the contribution, good job. I'm just leaving a comment with a little doubt.

File: test/v2/getEntry.test.js (hunk 1 above)
Comment (MrJuanGaviriaK) [reply test/v2/getEntry.test.js:15]: Hi @brayvasq , because in this "describe" we only have one individual test "IT", so the beforeEach block is not necessary.

Comment (brayvasq) [reply test/v2/getEntry.test.js:15]: @MrJuanGaviriaK What if we use in this case a [beforeAll](https://jestjs.io/docs/setup-teardown#one-time-setup) block?

File: test/v2/getEntry.test.js [hunk 2]
```diff
@@ -12,10 +12,7 @@ const getEntryUrl = 'https://www.tickspot.com/123456/api/v2/entries/123456.json'
 describe('getEntry', () => {
   describe('when API call is successful', () => {
```
Comment (brayvasq) [inline test/v2/getEntry.test.js:13]: @MrJuanGaviriaK Also, I think before this describe we need to clear the mocks using something like [clearAllMocks](https://jestjs.io/docs/jest-object#jestclearallmocks).  e.g:
```javascript
beforeEach(() => {
    jest.clearAllMocks();
});
```

File: test/v2/getEntry.test.js (hunk 1 above)
Comment (MrJuanGaviriaK) [reply test/v2/getEntry.test.js:15]: I'm going to check it !!!

File: test/v2/getEntry.test.js (hunk 2 above)
Comment (MrJuanGaviriaK) [reply test/v2/getEntry.test.js:13]: Thanks!!!

**Comment by MrJuanGaviriaK** [comment]
Hi @brayvasq 

The comments were solved.

Thanks!

**Review by brayvasq — APPROVED** [review]
